### PR TITLE
unittest, readme

### PR DESCRIPTION
**Pull request by ShuaibinLi**
- unittest
  + add py39 test env, remove py27
  + add A2C
- readme, requirements
  + parl 2.0.5->2.1.0
- tensorboard -> summary

### CODE REVIEW

File: benchmark/torch/ppo/README.md [hunk 1]
```diff
@@ -23,14 +23,14 @@ PARL currently supports the open-source version of Mujoco provided by DeepMind,
 ## How to use
 ### Mujoco-Dependencies:
 + python>=3.7+
-+ [parl>2.0.5](https://github.com/PaddlePaddle/PARL)
++ [parl>2.1.0](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline benchmark/torch/ppo/README.md]: parl>=2.1.0

File: parl/env/atari_wrappers.py [hunk 2]
```diff
@@ -27,6 +27,8 @@
 
 cv2.ocl.setUseOpenCL(False)
 
+__all__ = ['wrap_deepmind']
```
Comment (rical730) [inline parl/env/atari_wrappers.py]: 'wrap_deepmind', 'MonitorEnv', 'get_wrapper_by_cls' , refer to https://github.com/PaddlePaddle/PARL/blob/develop/examples/A2C/actor.py#L21

File: parl/env/compat_wrappers.py [hunk 3]
```diff
@@ -24,6 +24,8 @@
 V_STEP_CHANGED = '0.25.0'
 V_NPRANDOM_CHANGED = '0.21.0'
 
+__all__ = ['CompatWrapper']
```
Comment (rical730) [inline parl/env/compat_wrappers.py]: please include all global variable and `get_gym_version` , will be used in https://github.com/PaddlePaddle/PARL/blob/develop/parl/env/atari_wrappers.py#L24

File: parl/env/compat_wrappers.py [hunk 4]
```diff
@@ -36,6 +38,18 @@ def get_gym_version(version_str=gym.__version__):
...
 class CompatWrapper(gym.Wrapper):
+    """ Compatible for different versions of gym, especially for `step()` and `reset()` APIs.
+
+        .. code-block:: python
+        # old version of gym APIs
+        observation = env.reset()
+        observation, reward, terminated, truncated, info = env.step(action)
+        # new version of gym APIs
+        observation, info = env.reset()
+        observation, reward, done, info = env.step(action)
```
Comment (rical730) [inline parl/env/compat_wrappers.py]: Looks like you have mixed up old and new gym APIs. Maybe you can explain at the end of the doc that the new version of the gym env can be used in the same way as the old version of gym, after being wrapped by ComptWrapper

File: benchmark/torch/ppo/README.md (hunk 1 above)
Comment (ShuaibinLi) [reply benchmark/torch/ppo/README.md]: done

File: parl/env/atari_wrappers.py (hunk 2 above)
Comment (ShuaibinLi) [reply parl/env/atari_wrappers.py]: done

File: parl/env/compat_wrappers.py (hunk 3 above)
Comment (ShuaibinLi) [reply parl/env/compat_wrappers.py]: done

File: parl/env/compat_wrappers.py (hunk 4 above)
Comment (ShuaibinLi) [reply parl/env/compat_wrappers.py]: done

**Review by rical730 — APPROVED** [review]
LGTM~

**Pull request merged**